vertica/tests/common.py

```diff
@@ -20,4 +20,25 @@
     'password': 'monitor',
     'timeout': 10,
     'tags': ['foo:bar'],
+    'tls_verify': False,
 }
+
+# TLS certs
+CERTIFICATE_DIR = os.path.join(os.path.dirname(__file__), 'certificate')
+cert = os.path.join(CERTIFICATE_DIR, 'cert.cert')
+private_key = os.path.join(CERTIFICATE_DIR, 'server.pem')
+
+TLS_CONFIG = {
```

Comment:
Currently only used for `tls_verify`, but I will add another PR that has `use_tls`.